frontend/src/Buildings/actionCreator.js

```diff
@@ -9,10 +9,16 @@ export const ADD_BUILDING_FAILURE = 'addBuildingFailure';
 
 const URL = `http://${SERVER_URL}/kingdom/buildings`;
 
-export function fetchBuildings() {
+export function fetchBuildings(token) {
   return (dispatch) => {
     dispatch({ type: FETCH_BUILDINGS_REQUEST });
-    fetch(URL)
+    return fetch(URL, {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
```

Comment:
```suggestion
```

in a GET request you shouldn't set this header, because there is no content that you send